addOns/webdrivers/webdriverlinux/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 - Fix link in help page.
 
+## 8 - 2019-04-5
+
+- Update Help
```

Comment:
This can be removed, it's assumed that the help is kept up-to-date (only if adding more info would worth mentioning it).